packages/app/src/app/pages/Pro/Upgrade.tsx

```diff
@@ -162,6 +165,35 @@ export const ProUpgrade = () => {
       team.subscription?.paymentProvider === SubscriptionPaymentProvider.Paddle
   );
 
+  const getPricePerMonth = (
```

Comment:
The reason I'm keeping this function as a copy of the one in `UpsellTeamProCard.tsx` is that this might change in a relatively short time. We might want to show actual subscription prices to admins with an active subscription which will need some extra checks and changes in this function.